mitogen/buildah.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019, David Wilson
```

Comment:
This is literally just a slightly modified version of the Docker plugin so I just left your copyright.